Makefile

```diff
@@ -1,13 +1,16 @@
 GO_SRC = ./wc-go/src/wc.go
 GO_BIN = ./bin/wc-go
+RUST_SRC_DIR = ./wc-rust/
+RUST_SRC = ./wc-rust/src/main.rs
+RUST_BIN = ./bin/wc-rust
 
 HASKELL_SRC = ./wc-haskell/src/Lib.hs ./wc-haskell/app/Main.hs
 HASKELL_BIN = ./bin/wc-haskell
 
 .PHONY: all
-all: go haskell
+all: go haskell rust
 
-.PHONY: go
+.PHONY: go rust
```

Comment:
Nitpick, but could you please move this `.PHONY: rust` line down to above the `rust` target on line 27 for consistency?